tools/cli/admin.go

```diff
@@ -554,6 +554,10 @@ func newAdminClusterCommands() []cli.Command {
 					Usage:    "Remote cluster frontend address",
 					Required: true,
 				},
+				cli.BoolTFlag{
+					Name:  FlagConnectionEnableWithAlias,
+					Usage: "Optional: default ture. Enable remote cluster connection",
```

Comment:
Do you need to specify the value true for this flag if you want the default to be true?